### ISSUE TITLE
Mod page color contrast issues

### ISSUE THREAD

**Issue by stroudn1** [thepracticaldev/dev.to#7064]
**Describe the bug**

The color contrast on the majority of the mod page makes it hard to see/read

For hero: Text is #F9FAFA, background is #D4FFFE, contrast ratio is 1.03:1 (does not pass, should be at least 4.5)

Posts underneath: Text is #0A0A0A, background is #1A2634, contrast ratio is 1.29:1 (does not pass, should be at least 4.5)

**To Reproduce**

Go to https://dev.to/mod
See screenshots below for examples

**Expected behavior**

When color contrast is at 4.5:1 or higher, page elements are easy to see for visually impaired and non-impaired users. 

Hero (color recommendation): text color #707171 puts contrast at 4.6
Links (color recommendation): text color #8D8D8D puts contrast at 4.6

**Screenshots**
![image](https://user-images.githubusercontent.com/17433156/78366478-8f683b00-758e-11ea-9909-ac68e907403a.png)

![image](https://user-images.githubusercontent.com/17433156/78366541-a444ce80-758e-11ea-8ab7-bfc81d65608a.png)

**Desktop (please complete the following information):**

- OS: Windows 10
- Browser: Google Chrome
- Version: Version 80.0.3987.149 (Official Build) (64-bit)

**Comment by rhymes** [comment]
Hi @stroudn1, thanks for raising this issue. I'm going to close this a duplicate of https://github.com/thepracticaldev/dev.to/issues/5738 and let you know we'll get to it as soon as possible!
